start.py

```diff
@@ -1,10 +1,46 @@
 import sys
-
+import multiprocessing
+import argparse
 from com_window.main import comWindow
 from PyQt5 import QtWidgets
```

Comment:
```suggestion
from PyQt5 import QtWidgets
if getattr(sys, 'frozen', False) and sys.platform == 'darwin':
    matplotlib.use('Qt5Agg')
```

Can also be the TkAgg backend